miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -935,4 +942,64 @@ public static Library to(LibraryDto from, Library to) {
 
     return to;
   }
+
+  public static BoxDto asDto(Box from) {
+    BoxDto dto = new BoxDto();
+    dto.setId(from.getId());
+    dto.setAlias(from.getAlias());
+    dto.setDescription(from.getDescription());
+    dto.setIdentificationBarcode(from.getIdentificationBarcode());
+    dto.setUseId(from.getUse().getId());
+    dto.setUseAlias(from.getUse().getAlias());
+    dto.setSizeId(from.getSize().getId());
+    dto.setRows(from.getSize().getRows());
+    dto.setCols(from.getSize().getColumns());
+    dto.setScannable(from.getSize().getScannable());
+    dto.setItems(asBoxablesDtos(from.getBoxables()));
+    return dto;
+  }
+
+  public static List<BoxableDto> asBoxablesDtos(Map<String, Boxable> boxables) {
+    List<BoxableDto> items = new ArrayList<>();
+    for (Entry<String, Boxable> entry : boxables.entrySet()) {
+      items.add(asDto(entry.getValue(), entry.getKey()));
+    }
+    return items;
+  }
+
+  public static BoxableDto asDto(Boxable from) {
+    return asDto(from, from.getBoxPosition());
+  }
+
+  public static BoxableDto asDto(Boxable from, String position) {
+    BoxableDto dto = new BoxableDto();
+    dto.setPosition(position);
```

Comment:
BoxableDto doesn't have a `position` attribute; is this perhaps doubling up on the `setBoxPosition()` call below?